routes/team.js

```diff
@@ -28,4 +29,15 @@ team.post("/data", function(req, res) {
 	});
 });
 
-module.exports = team;
\ No newline at end of file
+team.post("/delete", authHelper.isAdmin, function(req, res) {
```

Comment:
I added an item to our Trello board to update our API to be more in line with REST practices.